src/main/java/lotto/domain/Money.java

```diff
@@ -1,29 +1,23 @@
 package lotto.domain;
 
 import lotto.exception.NotPositiveException;
-import org.apache.commons.lang3.StringUtils;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 public class Money {
 
-    private static int money;
+    private static final Pattern POSITIVE_PATTERN = Pattern.compile("^[0-9]*$");
+    private final int money;
```

Comment:
상수와 인스턴스 변수의 구분을 위해서 공백 라인을 추가해보면 좋을 것 같아요.
```suggestion
    private static final Pattern POSITIVE_PATTERN = Pattern.compile("^[0-9]*$");
    
    private final int money;
```